internal/config/exec.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"os/exec"
+	"runtime"
+)
+
+type Exec interface {
+	Cmd(commandLine string) bool
+}
+
+type osExec struct{}
+
+// NewOsExec returns an object that executes given commands in the OS.
+func NewOsExec() Exec {
+	return &osExec{}
+}
+
+// Cmd runs plain string command. It checks only exit code and returns bool value.
+func (o *osExec) Cmd(commandLine string) bool {
+	if commandLine == "" {
+		return false
+	}
+
+	var cmd *exec.Cmd
+	if runtime.GOOS == "windows" {
```

Comment:
@mrexox I added calling the command via `sh -c` but as you said, it may not work with `Windows` if it does not include WSL, Cygwin, and so on.

So I added checking the OS, but I couldn't check it because we don't have a running test-integrity for windows [here](https://github.com/evilmartians/lefthook/blob/master/.github/workflows/test.yml#L35)

I think there was a reason why it was not added to ci-cd for windows :)
But what do you think about checking the `windows` and adding test integrity for Windows to ci-cd?

Reply:
Let's try it out!

Reply:
@mrexox, Unfortunately, the `[windows] skip' command was necessary to skip some of the Windows test integrations.

Also, I use `PowerShell` for Windows commands because it supports more complex commands (e.g. with `if`).

We can avoid skipping some `test-integrities`, but it requires adapting those tests, and it is difficult without a Windows machine.

The example of skipping test [here](https://github.com/evilmartians/lefthook/pull/634/files#diff-3d168f7bd5c5e2a73b3bc1a841945beac68cb9960ba566d4716428a845c99981R1)

Reply:
Alright, I think this is acceptable for now. Thank you for digging into testscript, I appreciate it. I will try to figure out how to make it work for Windows when I have a chance to work on a Windows machine.